web_external/Studies/StudyView.js

```diff
@@ -48,5 +50,36 @@ isic.views.StudyView = isic.View.extend({
         });
 
         return this;
+    },
+
+    confirmDestroy: function () {
+        girder.confirm({
+            text: '<h4>Permanently delete <b>"' + _.escape(this.model.name()) + '"</b> study?</h4>',
+            escapedHtml: true,
+            confirmCallback: _.bind(function () {
+                // Ensure dialog is hidden before continuing. Otherwise,
+                // when destroy() displays its modal alert dialog,
+                // the Bootstrap-created element with class "modal-backdrop"
+                // is erroneously not removed.
+                $('#g-dialog-container').on('hidden.bs.modal', _.bind(this.destroyModel, this));
+            }, this)
+        });
+    },
+
+    destroyModel: function () {
+        this.model.destroy({
+            success: function (model, resp, options) {
+                isic.showAlertDialog({
+                    text: '<h4>Study <b>"' + _.escape(model.name()) + '"</b> deleted</h4>',
```

Comment:
I think we're inconsistent in whether we end text on alert dialogs with a period, but we should review that separately.